### ISSUE TITLE
Issue encountered at image push to blob storage 

### ISSUE THREAD

**Issue by MelihCelik00** [Azure/azure-storage-node#738]
> For latest features support, please switch to [Azure Storage JavaScript SDK V10](https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage). 

### Which service(blob, file, queue, table) does this issue concern?
Blob
### Which version of the SDK was used?
2.10.7
### What's the Node.js/Browser version?
v14.20.0
### What problem was encountered?
We use azure-storage for months now and this already-in-use service stopped working. Our image push (to blob storage) service which uses this deprecated, package instantly started to give errors even though nothing is touched in the component for months. "blobService.createBlockBlobFromText" is the main service that we use in our project. It does not return any error, code succeeds but image is not getting published.
### Steps to reproduce the issue?
Basically, we use this package in this flow:
```js
const util = require('util');

const blobService = azure.createBlobService(connectionString);
const blobCreateAsync = util.promisify(blobService.createBlockBlobFromText);
...
await blobCreateAsync.call(...)
```
### Have you found a mitigation/solution?
Not found any. Still searching for days.

**Comment by MelihCelik00** [comment]
I guess it's happened because of a difference happened in the validation packages, after I opened this issue I removed them and my problem of blob solved. Sorry if I took your time.
